server/chat-plugins/mafia.js

```diff
@@ -3084,6 +3120,19 @@ const commands = {
 			`/mafia playerroles - View all the player's roles in chat. Requires host`,
 			`/mafia end - End the current game of mafia. Requires host % @ * # & ~`,
 		].join('<br/>');
+		buf += `</details><details><summary class="button">IDEA Module Commands</summary>`;
```

Comment:
This is a good idea, though remember to remove these commands from the other categories (It should slim down host commands which is also nice).